docs/sources/writing-guide/front-matter/index.md

```diff
@@ -110,22 +107,25 @@ cascade:
       - cloud
 ```
 
-### title
+### menuTitle
+
+Use to specify a different heading in the sidebar navigation than the `title` element, for example if you want to abbreviate the topic heading in the table of contents.
+
+### (required) title
 
-**Required.**
+Hugo uses the `title` to generate the sidebar table of contents if there is no `menuTitle` specified in the front matter.
 
-Becomes the document title element. Often browsers display this in the tab for the page.
+The `title` becomes the document title element in the HTML. Often browsers display this in the tab for the page.
 
-It doesn't need to precisely match the `menuTitle`.
 Optimize the title for search engines. Use double quotes (`"`) to surround the title. Do not use smart quotes.
 
-If the `doc-validator` linter has been implemented on your repository, your topic heading must match the title in the metadata.
+If the `doc-validator` linter has been implemented on your repository, your topic heading must exactly match the title in the metadata.
 
 ### weight
 
-Determines the placement of the topic within the left-hand sidebar on https://grafana.com. Smaller numbers place the topic higher in the guide. Pages with the same weight have lexicographic ordering.
+You use the `weight` to specify the placement of a topic within the left-hand sidebar on https://grafana.com. Smaller numbers place the topic higher in the guide or section of the guide. Pages with the same weight or no weight will be displayed in alphabetical order.
```

Comment:
```suggestion
Use the `weight` front matter to specify the placement of a topic within the left-hand sidebar on https://grafana.com.
Smaller numbers place the topic higher in the guide or section of the guide.
Pages with the same weight will be displayed in alphabetical order.
```

No weight is not an exceptional case, it gets the default weight value and all pages with the same weight are sorted alphabetically.

Reply:
Hmmm, I need to figure out how to reword that.  What I was aiming for was that if you don't specify weights at all that pages are displayed in alphabetical order.  What is the default weight value?  I didn't realize there was one?

Reply:
I believe the default is 0.

Reply:
I double checked because I wasn't certain. It turns out I was completely wrong, sorry about that. According to the maintainer, pages are sorted by `weight` only if it set and then by `linkTitle` or `title`. So In that case, I think your update is better than my suggestion.

https://discourse.gohugo.io/t/how-do-i-sort-by-weight-then-title/18297/2